controller/scan/jfrog.go

```diff
@@ -232,7 +249,12 @@ func (r *jfrog) GetTagList(domain, repo, tag string) ([]string, error) {
 		}
 		repo = subRepo
 	}
-	return rc.Tags(repo)
+
+	if tags, err := rc.Tags(repo); len(tags) > 0 {
+		return tags, err
+	}
```

Comment:
Shall we check the error as well?

Reply:
I don't think so. We can log this as a debug message; it typically just return the error and let the outer function handle it.

@holyspectral what do you think?

Reply:
My point is that when `len(tags) == 0`, the `err` will be ignored.  Errors should be handled all the time because it can be something unexpected, where fallback might not be appropriate.

Reply:
I see, just add new error check